src/js/components/search/newResultsView/SectionsContent.jsx

```diff
@@ -33,17 +32,13 @@ const SectionsContent = (props) => {
         entries.forEach((entry) => {
             const section = entry.target.className;
             if (entry.isIntersecting) {
-                if (section === 'award') {
+                if (section === 'awards') {
                     setAwardTableHasLoaded(true);
-                    console.log("award");
+                    console.log("awards");
```

Comment:
ik these were for troubleshooting but now that progressive loading works do we still need them